### PR TITLE
update references to poetry master branch

### PR DESCRIPTION
**Pull request by abn**

### CODE REVIEW

File: content/blog/2024-02-25-announcing-poetry-1-8-0.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ has to download the whole file to calculate the hash itself.
 Although we tested this feature with various package indices,
 there still might be servers that respond to HTTP range requests in an unexpected way.
 Therefore, if you experience any issues with this feature,
-you can [disable lazy-wheel](docs/master/configuration#solverlazy-wheel).
+you can [disable lazy-wheel](docs/main/configuration#solverlazy-wheel).
```
Comment (radoering) [inline content/blog/2024-02-25-announcing-poetry-1-8-0.md:69]: This link is a 404 in the preview. Is this just a preview issue?

Comment (abn) [reply content/blog/2024-02-25-announcing-poetry-1-8-0.md:69]: Something is off, looking.

**Comment by radoering** [comment]
Will it be a problem that we have both?

https://github.com/python-poetry/website/blob/1fd2c91c4018fc18e0b71017b53b39772753dbcb/bin/website#L49-L54

File: content/blog/2024-02-25-announcing-poetry-1-8-0.md (hunk 1 above)
Comment (abn) [reply content/blog/2024-02-25-announcing-poetry-1-8-0.md:69]: It is also 404 on live version the rel link is wrong I think.

```
https://python-poetry.org/blog/announcing-poetry-1.8.0/docs/master/configuration#solverlazy-wheel
```

Comment (radoering) [inline content/blog/2024-02-25-announcing-poetry-1-8-0.md:69]: Ooops. Maybe, just

```suggestion
you can [disable lazy-wheel](docs/configuration#solverlazy-wheel).
```

Comment (abn) [reply content/blog/2024-02-25-announcing-poetry-1-8-0.md:69]: It also needs a `/` in the front.

**Review by radoering — APPROVED** [review]

**Pull request merged**